src/basic/cgroup-util.c

```diff
@@ -149,6 +149,20 @@ bool cg_ns_supported(void) {
         return enabled;
 }
 
+bool cg_freezer_supported(void) {
```

Comment:
Oh, this should go in shared/, not /basic/.

Reply:
why? sounds generic enough and similar enough to many other cg_xyz() checkers to stay here...